scripts/examples/typescript/tests/getXapiVersion.ts

```diff
@@ -0,0 +1,28 @@
+import { xapi_client } from "../src/XenAPI"
+
+// Ref https://xapi-project.github.io/xen-api/
+async function get_api_version(session: any) {
+    const pool = await session.xenapi.pool.get_all()
```

Comment:
This can be achieved in a faster way with 2 requests instead of 4: call `pool.get_all_records`, then call `host.get_record` for pool[0].master and then user the fields API_version_major/minor of the host record.